portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,81 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Returns an array of comments in JSON form */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  int maxComments=3;
```

Comment:
nit: I'd put whitespace so that it looks like `int maxComments = 3` for things like this and other places below. See https://google.github.io/styleguide/javaguide.html#s4.6-whitespace for additional details

Since this appears to be a constant, it should probably look like `static final int MAX_COMMENTS = 3` according to https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Also, should this be a private variable?

Reply:
Sorry, I'll the name because this is not a constant (not a maximum number). I did make it a private variable.

Reply:
*I'll change the name